proposals/network_canister_management/ledger_20211117T2319Z.md

```diff
@@ -0,0 +1,124 @@
+## Summary
+
+Updating Ledger to Wasm module with hash `61a312e4bf4f86899edabf6b3cedc504b2b0825d9b7956abea65582095692a9a` built from git commit `88d24e7def3b6ef41508d753f4037dd76d940a6d`.
+
+## Rationale
+
+This update contains changes required to allow all canisters to transfer ICP by sending messages to the Ledger canister:
+
+  * The ledger doesn't apply the whitelist when it decides whether to allow the transfer to the caller.
+    All principals except for the anonymous are allowed to transfer ICP now.
+
+  * The ledger exposes 2 new Candid endpoints: `account_balance` and `transfer` (see the full Candid interface below).
+    More Candid endpoints (e.g., an endpoint for fetching blocks) shall be added in subsequent proposals.
+
+The newly exposed candid interface:
+
+```candid
+// Amount of ICP tokens, measured in 10^-8 of a token.
+type ICP = record {
+     e8s : nat64;
+};
+
+// Number of nanoseconds from the UNIX epoch in UTC timezone.
```

Comment:
```suggestion
// Number of nanoseconds from the UNIX epoch (00:00:00 UTC, Jan 1, 1970).
```

Reply:
> nanosecs is not real time but IC time

Not all timestamps here are IC time: the transaction `created_at_time` timestamp is chosen by the client, it's not necessary IC time. Furthermore, there is no globally synchronized "IC time", different subnets use different clocks. So it's quite possible that transaction.create_at_time will have a timestamp that is larger than the timestamp of the block it's included into.

Reply:
Ok, updated suggestion to just specify the epoch (the nitpick being that UNIX epoch is the same in all timezones)